Bio/GenBank/Scanner.py

```diff
@@ -505,6 +509,7 @@ def parse_cds_features(self, handle,
             self.parse_header()  # ignore header lines!
             feature_tuples = self.parse_features()
             # self.parse_footer() # ignore footer lines!
+            line = ""
```

Comment:
I would have said that was redundant (or is this one of those subtleties about scope in Python that sometimes catches me out?)